5-network/10-long-polling/article.md

```diff
@@ -38,31 +38,31 @@ If the connection is lost, because of, say, a network error, the browser immedia
 A sketch of client-side `subscribe` function that makes long requests:
 
 ```js
-async function subscribe() {
-  let response = await fetch("/subscribe");
+async function subscribe(url) {
+  let response = await fetch(url);
```

Comment:
Since we don't changing `response` it below, let's use `const`:

```suggestion
  const response = await fetch(url);
```

Reply:
Thanks! I suppose in a small scope like this, we don't have to be "const-correct" like what C++ taught me quite a few years ago. Other parts of the Tutorial seem to use the same strategy. The "When to Use const" note in David Flanagan's JavaScript: The Definitive Guide (7th edition, p. 54) is also a good read.